client/lib/plans/constants.js

```diff
@@ -752,6 +842,130 @@ export const FEATURES_LIST = {
 			'Custom polls, surveys, ratings, and quizzes for the ultimate in customer and reader engagement.'
 		)
 	},
+
+	[ FEATURE_CORE_JETPACK ]: {
+		getSlug: () => FEATURE_CORE_JETPACK,
+		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getDescription: () => i18n.translate(
+			'Stats, themes, and promotion tools.'
+		),
+		hideInfoPopover: true
+	},
+	[ FEATURE_BASIC_SECURITY_JETPACK ]: {
+		getSlug: () => FEATURE_BASIC_SECURITY_JETPACK,
+		getTitle: () => i18n.translate( 'Basic Security' ),
+		getDescription: () => i18n.translate(
+			'Brute force protection, monitoring, secure logins, updates.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_BASIC_SUPPORT_JETPACK ]: {
+		getSlug: () => FEATURE_BASIC_SUPPORT_JETPACK,
+		getTitle: () => i18n.translate( 'Basic Support' ),
+		getDescription: () => i18n.translate(
+			'Free support to help you make the most of Jetpack.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_SPEED_JETPACK ]: {
+		getSlug: () => FEATURE_SPEED_JETPACK,
+		getTitle: () => i18n.translate( 'Speed and Storage' ),
+		getDescription: () => i18n.translate(
+			'Unlimited use of our high speed image content delivery network.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_SPEED_ADVANCED_JETPACK ]: {
+		getSlug: () => FEATURE_SPEED_ADVANCED_JETPACK,
+		getTitle: () => i18n.translate( 'Speed and Storage' ),
+		getDescription: () => i18n.translate(
+			'Also includes 13Gb of high-speed, ad-free video hosting.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_SPEED_UNLIMITED_JETPACK ]: {
+		getSlug: () => FEATURE_SPEED_UNLIMITED_JETPACK,
+		getTitle: () => i18n.translate( 'Speed and Storage' ),
+		getDescription: () => i18n.translate(
+			'Also includes unlimited, high-speed, ad-free video hosting.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_SITE_BACKUPS_JETPACK ]: {
+		getSlug: () => FEATURE_SITE_BACKUPS_JETPACK,
+		getTitle: () => i18n.translate( 'Site Backups' ),
+		getDescription: () => i18n.translate(
+			'Automated daily backups (unlimited storage), one-click restores, and 30-day archive.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_SECURITY_SCANNING_JETPACK ]: {
+		getSlug: () => FEATURE_SECURITY_SCANNING_JETPACK,
+		getTitle: () => i18n.translate( 'Advanced Security' ),
+		getDescription: () => i18n.translate(
+			'Also includes daily scans for malware and security threats.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_REVENUE_GENERATION_JETPACK ]: {
+		getSlug: () => FEATURE_REVENUE_GENERATION_JETPACK,
+		getTitle: () => i18n.translate( 'Revenue Generation' ),
+		getDescription: () => i18n.translate(
+			'High quality ads to generate income from your site.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Display high-quality ads to generate income directly from your site.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).